src/main/native/com/code_intelligence/jazzer/android/BUILD.bazel

```diff
@@ -0,0 +1,71 @@
+load("//bazel:compat.bzl", "SKIP_ON_WINDOWS")
+
+genrule(
+    name = "android_jvmti_h_gen",
+    outs = ["jvmti.h"],
+    cmd = "curl --silent https://android.googlesource.com/platform/art/+/1cff8449bac0fdab6e84dc9255c3cccd504c1705/openjdkjvmti/include/jvmti.h?format=TEXT | base64 --decode > $(OUTS)",
+    target_compatible_with = SKIP_ON_WINDOWS,
+)
+
+cc_library(
+    name = "genrule_builder",
+    data = [":android_jvmti_h_gen"],
+    linkstatic = 1,
+    tags = ["manual"],
+    target_compatible_with = SKIP_ON_WINDOWS,
+)
+
+cc_binary(
+    name = "native_agent",
+    srcs = [
+        "dex_file_manager.cpp",
+        "dex_file_manager.h",
+        "jazzer_jvmti_allocator.h",
+        "jvmti.h",
+        "native_agent.cpp",
+    ],
+    copts = [
+        "-Wno-unused-variable",
+    ],
+    includes = [
+        ".",
+    ],
+    linkopts = [
+        "-lz",
+    ],
+    linkshared = True,
+    tags = ["manual"],
+    target_compatible_with = SKIP_ON_WINDOWS,
+    deps = [
+        ":genrule_builder",
+        "@com_google_absl//absl/strings",
+        "@jazzer_slicer",
+    ],
+)
+
+android_library(
+    name = "native_agent_android_wrapper",
+    data = [
+        ":android_jvmti_h_gen",
```

Comment:
As above, can the `data` attribute be added to the `android_binary` target below instead?